lib/Runtime/Language/JavascriptOperators.cpp

```diff
@@ -1091,10 +1090,30 @@ namespace Js
             }
             else
             {
-                return object && object->HasOwnProperty(propertyId);
+#ifdef DEBUG
+                // In debug builds, calculate expectedAnswer up-front so that
+                // we can Assert the fastpath gets the right result.
+                BOOL expectedAnswer = object && object->HasOwnProperty(propertyId);
+#endif
+
+                PropertyString *propString = requestContext->GetPropertyString(propertyId);
```

Comment:
Great catch.

---
In reply to: [75036981](https://github.com/Microsoft/ChakraCore/pull/1449#discussion_r75036981) [](ancestors = 75036981)